tox.ini

```diff
@@ -1,5 +1,5 @@
 [tox]
-envlist = py26,py27,py33,py34,py35
+envlist = py27,py34,py35
```

Comment:
Dropped per [pytest](https://docs.pytest.org/en/latest/changelog.html#pytest-3-3-0-2017-11-23). Not ready to add 3.6 yet because tests fail (debug opcode drift?).